typings/index.d.ts

```diff
@@ -1,89 +1,112 @@
-import * as React from 'react';
```

Comment:
I guess `kcd-scripts` is working 😅 

We're now applying prettier to `ts` files! :tada: